<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HyperSpectralViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QWidget" name="widgetCentral" native="true">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>256</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="viewerLayout"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderLayers">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_view">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>200</width>
           <height>570</height>
          </rect>
         </property>
         <attribute name="label">
          <string>View</string>
         </attribute>
         <widget class="QPushButton" name="buttonRgbImage">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Show RGB image</string>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonShowClusters">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>300</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Show Clusters image</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_R_channel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>R channel</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_G_channel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>G channel</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_B_channel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>B channel</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineR">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>0</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineG">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineB">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>60</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textBoxClusters">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>181</width>
            <height>71</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineThreshold">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>270</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>0.95</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_Threshold">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>270</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Threshold</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_Metric_selection">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>240</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <item>
           <property name="text">
            <string>angle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>pearson</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>module</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>chebyshev</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>240</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Metric</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>520</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>angle: 0.9 ~ 0.99</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>530</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>pearson: 0.9 ~ 0.99</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>540</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>module: 8000 ~ 10000</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>550</y>
            <width>141</width>
            <height>21</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>chebyshev: 2500 ~ 4000</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_use_reference">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>340</y>
            <width>171</width>
            <height>20</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>(WiP) Use available reference</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_quan_of_class">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>360</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Quantity of classes:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_show_layer">
          <property name="geometry">
           <rect>
            <x>6</x>
            <y>550</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>layer: 0</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_canny">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>210</y>
            <width>70</width>
            <height>17</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Canny</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_num_class">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>380</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Сlass number:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_num_of_pix">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>400</y>
            <width>181</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Number of pixels in the class:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_bac_to_img">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>430</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Back to image</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_show_hist">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>430</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Show hist</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_classification">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>200</width>
           <height>570</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Classification</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_program"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionSignatures_window"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionAbout_program">
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="actionSignatures_window">
   <property name="text">
    <string>Signatures window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
